src/modules/universal-discovery/components/menu-button/menu.button.js

```diff
@@ -20,13 +20,15 @@ MenuButton.propTypes = {
     extraClasses: PropTypes.string,
     onClick: PropTypes.func.isRequired,
     isDisabled: PropTypes.bool,
+    title: PropTypes.string,
     children: PropTypes.any,
 };
 
 MenuButton.defaultProps = {
     children: [],
     extraClasses: '',
     isDisabled: false,
+    title: '',
```

Comment:
Are you sure it should be empty string instead of null/undefined? Wouldn't it create empty tooltip?

Reply:
it's ok because tooltips are create only for not empty title